main.go

```diff
@@ -52,6 +54,8 @@ func (c *rootCmd) RegisterFlags(fl *flag.FlagSet) {
 	fl.BoolVar(&c.skipSync, "skipsync", false, "skip syncing local settings and extensions to remote host")
 	fl.BoolVar(&c.syncBack, "b", false, "sync extensions back on termination")
 	fl.BoolVar(&c.printVersion, "version", false, "print version information and exit")
+	fl.BoolVar(&c.noOpen, "no-open", false, "do not open web browser")
```

Comment:
after speaking with a colleague let's handle the `no-open` in a separate PR, sorry for indicating otherwise earlier

Reply:
No worries. WIll change it